### PR TITLE
cnf-network: Added logLevel to SriovNetworks

### PR DESCRIPTION
**Pull request by evgenLevin**

### CODE REVIEW

File: tests/cnf/core/network/dpdk/tests/rootless.go [hunk 1]
```diff
@@ -652,7 +652,7 @@ func defineAndCreateVlanNad(name, intName string, vlanID uint16, ipam *nad.IPAM)
 func defineAndCreateSrIovNetwork(srIovNetwork, resName string, vlanID uint16) {
 	srIovNetworkObject := sriov.NewNetworkBuilder(
 		APIClient, srIovNetwork, NetConfig.SriovOperatorNamespace, tsparams.TestNamespaceName, resName).
-		WithMacAddressSupport()
+		WithMacAddressSupport().WithLogLevel("debug")
```
Comment (gkopels) [inline tests/cnf/core/network/dpdk/tests/rootless.go:655]: Do we want to create a variable for "debug"?

Comment (evgenLevin) [reply tests/cnf/core/network/dpdk/tests/rootless.go]: changed

**Review by ajaggapa — APPROVED** [review]
lgtm

**Comment by gkopels** [comment]
lgtm

**Review by gkopels — APPROVED** [review]

**Pull request merged**